newapi/ooniapi/measurements.py

```diff
@@ -719,29 +836,82 @@ def list_measurements():
         if until is None:
             if report_id is None:
                 until = date.today() + timedelta(days=1)
-        else:
-            until = parse_date(until)
     except ValueError:
         raise BadRequest("Invalid until")
 
     try:
         if since is None:
             if report_id is None and until is not None:
                 since = until - timedelta(days=30)
-        else:
-            since = parse_date(since)
     except ValueError:
         raise BadRequest("Invalid since")
 
     if order.lower() not in ("asc", "desc"):
         raise BadRequest("Invalid order")
 
+    if current_app.config["USE_CLICKHOUSE"]:
+        return _list_measurements_click(
+            since,
+            until,
+            report_id,
+            probe_cc,
+            probe_asn,
+            test_name,
+            anomaly,
+            confirmed,
+            failure,
+            input_,
+            domain,
+            category_code,
+            order,
+            order_by,
+            limit,
+            offset,
+        )
+    else:
+        return _list_measurements_pg(
+            since,
+            until,
+            report_id,
+            probe_cc,
+            probe_asn,
+            test_name,
+            anomaly,
+            confirmed,
+            failure,
+            input_,
+            domain,
+            category_code,
+            order,
+            order_by,
+            limit,
+            offset,
+        )
+
+
+def _list_measurements_pg(
+    since,
+    until,
+    report_id,
+    probe_cc,
+    probe_asn,
+    test_name,
+    anomaly,
+    confirmed,
+    failure,
+    input_,
+    domain,
+    category_code,
+    order,
+    order_by,
+    limit,
+    offset,
+):
     INULL = ""  # Special value for input = NULL to merge rows with FULL OUTER JOIN
```

Comment:
This can be deleted now I guess.

Reply:
No, why? It's the Postgres version of the query.

Reply:
I didn't notice that it was being used further down, I thought it was no longer used since it was dropped from the query itself.